### DISCUSSION TITLE
Welcome to sc-im Discussions!

### DISCUSSION THREAD

**Discussion by andmarti1424** [andmarti1424/sc-im#500] [General]
## 👋 Welcome to SC-IM community!
We (just I) are a little full with issues. Please comment on which is the most important bug to fix first and/or the feature request most needed.
Please share your ideas!
THANK YOU

**Comment by KirillSapelkin** [comment]
Hello,

I'm comming from sc spreadshet.

There, when changes made to the configuration of are 
saved with the spreadsheet.  In sc-im I noticed that color
changes and changes such are numeric or nonumeric are lost
when the spreadsheet is saved.

I prefer the way it is done in sc.

Could or should such a change be made to sc-im?
Perhaps, in scimrc, giving the user a choice which changes should be saved and 
which should not?


Kirill Sapelkin
